network/ssh/reachable.go

```diff
@@ -67,7 +67,7 @@ type hostKeyChecker struct {
 	// HostPort is the identifier that corresponds to this connection
 	HostPort network.HostPort
 
-	// Accepted will be passed HostPort if it validated the connection
+	// Accepted will be passed SpaceHostPort if it validated the connection
```

Comment:
I'm a bit confused by this comment.  Accepted a HostPort, not a SpaceHostPort.  Is "passed" equivalent to "given during creation"?  Otherwise it's an empty struct?